chatbot_ner/setup_logs.py

```diff
@@ -0,0 +1,114 @@
+import collections
```

Comment:
Whenever we add a new file it's good to add a module level docstring to indicate the purpose of the new module, even if it is obvious.
It should include basic usage instructions.

Reply:
doc string added `This module contains functions and classes required for configuring structlog`